spec/features/03_done_spec.rb

```diff
@@ -0,0 +1,200 @@
+    
+    within(:css, "form") do
+      fill_in "Email", with: user_jacob.email
+      fill_in "Password", with: user_jacob.password
+      find("button", :text => /Sign in/i ).click
+    end
+
+    visit("/")
+
+    expect(page).to have_css("div.done"),
+      "Expected to find a <div> element with the a class attribute containing 'done', but didn't find one."
+  end
+end
+
+describe "The Done section" do
+  it "has the color of all text styled as grey",js: true, points: 1 do
+    visit("/user_sign_in")
+    user_jacob = User.new
+    user_jacob.email = "jacob_#{rand(100)}@example.com"
+    user_jacob.password = "password"
+    user_jacob.save
+
+    visit "/user_sign_in"
+    
+    within(:css, "form") do
+      fill_in "Email", with: user_jacob.email
+      fill_in "Password", with: user_jacob.password
+      find("button", :text => /Sign in/i ).click
+    end
+
+    visit("/")
+    done_section_div = find("div.done")
+
+    # done_section_div.assert_matches_style( 'color' => 'grey')
+    expect(done_section_div).to have_color("grey")
+  end
+end
+
+describe "The Done section" do
+  it "displays todo items in a <li> element", points: 1 do
+    visit("/user_sign_in")
+    user_jacob = User.new
+    user_jacob.email = "jacob_#{rand(100)}@example.com"
+    user_jacob.password = "password"
+    user_jacob.save
+
+    visit "/user_sign_in"
+
+    within(:css, "form") do
+      fill_in "Email", with: user_jacob.email
+      fill_in "Password", with: user_jacob.password
+      find("button", :text => /Sign in/i ).click
+    end
+
+    visit("/")
+    
+    within(:css, "form") do
+      fill_in "Describe the task:", with: "Work on Todo List app"
+      find("button", :text => /Add to Next Up/i ).click
+    end
+    
+    visit("/")
+
+    within(:css, "div.next_up") do
+      within(:css, "form") do
+        find("button", :text => /Move to In Progress/i ).click
+      end
+    end
+
+    visit("/")
+
+    within(:css, "div.in_progress") do
+      within(:css, "form") do
+        find("button", :text => /Move to Done/i ).click
+      end
+    end
+
+    expect(page).to have_tag("div.done") do     
+      with_tag("ul") do
+        with_tag("li", text: /Work on Todo List app/i)
+      end
+    end
+  end
+end
+
+describe "The Done section" do
+  it "displays the formatted updated at time for each todo items", points: 1 do
+    visit("/user_sign_in")
+    user_jacob = User.new
+    user_jacob.email = "jacob_#{rand(100)}@example.com"
+    user_jacob.password = "password"
+    user_jacob.save
+
+    visit "/user_sign_in"
+
+    created_at = Time.now
+
+    within(:css, "form") do
+      fill_in "Email", with: user_jacob.email
+      fill_in "Password", with: user_jacob.password
+      find("button", :text => /Sign in/i ).click
+    end
+
+    visit("/")
+
+    within(:css, "form") do
+      fill_in "Describe the task:", with: "Work on Todo List app"
+      find("button", :text => /Add to Next Up/i ).click
+    end
+
+    updated_at = nil
```

Comment:
This may not be necessary due to L133 below?

Reply:
It appears that it's necessary. I assume because of the scope created with `travel`.

Without this line I get this error
![image](https://user-images.githubusercontent.com/17581658/199744740-1c897ad2-1ae3-449d-b4b8-a00b1dca5efb.png)